zeppelin-web/bower.json

```diff
@@ -35,7 +35,8 @@
     "select2": "^4.0.3",
     "github-markdown-css": "^2.4.0",
     "MathJax": "2.7.0",
-    "ngclipboard": "^1.1.1"
+    "ngclipboard": "^1.1.1",
+    "angular-viewport-watch": "^0.1.35"
```

Comment:
It would be nicer to install `angular-viewport-watch` using npm instead of bower. Since we are trying to move npm from bower.

Here is one example.

- https://github.com/apache/zeppelin/pull/2327